server/memphis_handlers_stations.go

```diff
@@ -1747,7 +1747,7 @@ func (s *Server) useSchemaDirect(c *client, reply string, msg []byte) {
 		return
 	}
 	if !exist {
-		errMsg := "Schema " + schemaName + " does not exist" + err.Error()
+		errMsg := "Schema " + schemaName + " does not exist"
```

Comment:
remove the error since it is nil in this case, does not exist is the actual error